terraform/helm/genesis/templates/NOTES.txt

```diff
@@ -0,0 +1,11 @@
+Your {{ .Chart.Name }} deployment named {{ .Release.Name }} is now deployed.
+
+To check automated genesis ceremony status:
+
+    $ kubectl logs -f job/{{ include "aptos-genesis.fullname" . }}-e{{ .Values.chain.era }}
+
+The resulting genesis ConfigMaps can be mounted by aptos-node, and will be located at:
+
+{{- range $user := .Values.genesis.users }}
+    - {{ $user.name }}-genesis-e{{ $.Values.chain.era }}
```

Comment:
what's the `-e` for? typo?

Reply:
The `-e<NN>` denotes the `era`, which we use to name a few things so we can wipe chain state.

(e.g. see the docs here https://aptos.dev/tutorials/validator-node/run-validator-node-using-aws)